lsql/judge/templates/problem.html

```diff
@@ -53,6 +53,19 @@ <h2>Resultado esperado</h2>
 {% block solution %}
 <h2>Solución</h2>
   <script src="{% static 'judge/js/submit.js' %}"></script>
+  <form onchange="javascript:load_submission_code(event);" class="mb-3">
+	<div class="custom-file">
+		<input type="file" class="custom-file-input" id="customFile" lang="es">
+        <label class="custom-file-label" for="customFile">Elige un archivo</label>
+    </div>
+    <script>
+		// The name of the file appear on select
+		$(".custom-file-input").on("change", function() {
+			var fileName = $(this).val().split("\\").pop();
+			$(this).siblings(".custom-file-label").addClass("selected").html(fileName);
```

Comment:
El código `$(this).siblings(".custom-file-label")` es únicamente para obtener el elemento `<label class="custom-file-label"`, ¿no? ¿Por qué no lo encuentras directamente con `$( ".custom-file-label" )`?

Reply:
Sí, es solo para eso. Los ejemplos que vi ponían eso y lo dejé por si acaso era una mejor forma, lo he quitado igualmente ahora y funciona bien.

Reply:
Principalmente era porque no sabía por qué buscabas desde los siblings cuando tenías una clase personalizada. ¿Quizá es más rápido? En cualquier caso ambos deberían funcionar igual.